arch/arm/boot/dts/overlays/revpi-flat-overlay.dts

```diff
@@ -117,30 +117,12 @@
 				brcm,function = <BCM2835_FSEL_GPIO_OUT>;
 				brcm,pull     = <BCM2835_PUD_OFF>;
 			};
-			tpm_pins {
```

Comment:
The irq pin is used by the tpm node. You should add a reference from there to this pinctrol. The reset is currently unused by the tpm node. If the driver supports a reset gpio we could add it here.